app/views/follow_requests/create.js.erb

```diff
@@ -0,0 +1,3 @@
+console.log("i clicked follow")
+$(".<%= dom_id(@follow_request.sender) %>_<%= dom_id(@follow_request.recipient)%>_follow_unfollow").replaceWith("<%=j(render 'follow_requests/follow_unfollow', sender: @follow_request.sender, recipient: @follow_request.recipient)%>");
```

Comment:
this is great! remember to assign your jquery to variables so that your code is clearer to read!